### PR TITLE
Localize web errors and improve error messages

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

- Localized all WebErrors message used in forms.
- Added a function to convert snake cased strings to camel case.
- Improved overall error messages displayed in the Identities' forms.
- Removed some of the hard coded strings in e2e tests.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-9r2ghq [refato] - Localize WebErrors message](https://app.clickup.com/t/9r2ghq)

File: content/locales/en.ts [hunk 1]
```diff
@@ -172,5 +172,10 @@ export const en = {
       "{characterCount, plural, one {1 character} other {{characterCount} characters}}",
     somethingWrong: "Something went wrong. Please try again later.",
     invalidValidationCode: "Invalid validation code.",
+    identityInputCantBeBlank: "Please fill in the following field.",
```
Comment (Oluane) [inline content/locales/en.ts:175]:
```suggestion
    identityInputCantBeBlank: "Please fill in the following field: ",
```

It's just a suggestion, I'm not sure what's the best between the two (same goes for the corresponding key in fr translations)

Comment (Fuzznimp) [reply content/locales/en.ts]: I made the changes :).

**Review by Oluane — APPROVED** [review]
Nice job 🙂

**Pull request merged**